src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Components/ApiVersionHelper.cs

```diff
@@ -26,6 +22,11 @@ namespace Microsoft.Azure.Commands.ResourceManager.Cmdlets.Components
     using System.Threading;
     using System.Threading.Tasks;
 
+    using Entities.Providers;
```

Comment:
Visual Studio prefer "System.xxx" on top and then other using in alphabet order.